src/JobHelper.groovy

```diff
@@ -79,6 +79,22 @@ class JobHelper {
     * Queries the Adopt API for all releases
     * @return
     */
-    public static getAvailableReleases() { return queryApi("https://api.adoptopenjdk.net/v3/info/available_releases") }
+    public static getAvailableReleases() {
+        def responseObject = null
+        for (int count = 0; count < 5; count++) {
+            try {
+                responseObject = queryApi("https://api.adoptopenjdk.net/v3/info/available_release")
+                break
+            } catch (RuntimeException e) {
+                println "Failed to query or parse the adopt api\nError:${e}\nRetrying in 30 seconds..."
+                sleep 30
```

Comment:
does sleep not need to be imported?

Reply:
and also make sure that is 30 seconds not milliseconds

Reply:
It shouldn't be. Sleep is used [elsewhere](https://github.com/AdoptOpenJDK/openjdk-build/blob/master/pipelines/build/common/config_regeneration.groovy#L293) without an import

Reply:
> and also make sure that is 30 seconds not milliseconds

Done